openflow13/nxext_test.go

```diff
@@ -14,7 +15,7 @@ func TestNXActionResubmit(t *testing.T) {
 	action1 := NewNXActionResubmit(portID)
 	data1, err := action1.MarshalBinary()
 	if err != nil {
-		t.Errorf("Failed to invoke NXAST_RESUBMIT_TABLE.MarshalBinary: %v", err)
+		t.Fatalf("Failed to invoke NXAST_RESUBMIT_TABLE.MarshalBinary: %v", err)
 	}
 	testAction1 := new(NXActionResubmitTable)
```

Comment:
I am a bit confused, why isn't this `newNXActionResubmitTable` like for all subsequent cases?